dev/com.ibm.ws.concurrent.persistent_fat_delayexec/publish/files/features/testFeature-1.0.mf

```diff
@@ -0,0 +1,8 @@
+Subsystem-ManifestVersion: 1
+Subsystem-SymbolicName: testFeature-1.0; visibility:=public
+Subsystem-Version: 1.0
+Subsystem-Content: test.concurrent.persistent.delayexec;  version="[1,1.0.100)",
```

Comment:
it looks like this manifest was modified from the original FAT a bit. If you pull over the bnd file for the user feature (see other comment) then you shouldn't have to modify this user feature manifest at all